README.md

```diff
@@ -1,4 +1,4 @@
-**This repo will be archived during _October_ as it is for practice only**
+﻿**This repo will be archived during _October_ as it is for practice only**
```

Comment:
What changes were made in this line?
The diff displays some changes, but I don't see any changes in the text

Reply:
nothing. i did nothing here